cpp/src/copying/copy.cpp

```diff
@@ -77,7 +78,17 @@ std::unique_ptr<column> allocate_like(column_view const& input,
 std::unique_ptr<column> empty_like(column_view const& input)
 {
   CUDF_FUNC_RANGE();
-  return make_empty_column(input.type());
+
+  std::vector<std::unique_ptr<column>> children;
+  auto child_iter = thrust::make_transform_iterator(
```

Comment:
Mithun added a `child_begin/child_end` in the struct column PR. That'd be nicer here.

Reply:
Swiped!